hack/consecutive-upgrades-test.sh

```diff
@@ -0,0 +1,259 @@
+        ${CMD} -n "${HCO_NAMESPACE}" patch installPlan "${INSTALL_PLAN}" --type merge --patch '{"spec":{"approved":true}}'
+        INSTALL_PLAN_APPROVED=true
+        break
+      fi
+      sleep 5
+  done
+
+  [[ "${INSTALL_PLAN_APPROVED}" = true ]]
+
+  ## Verify the subscription has changed to the new version
+  #  currentCSV: kubevirt-hyperconverged-operator.v100.0.0
+  #  installedCSV: kubevirt-hyperconverged-operator.v100.0.0
+  Msg "Verify the subscription's currentCSV and installedCSV have moved to the new version"
+
+  ${CMD} get pods -n ${HCO_NAMESPACE}
+  ./hack/retry.sh 30 60 "${CMD} get deployment -n ${HCO_NAMESPACE} | grep ${HCO_DEPLOYMENT_NAME}"
+
+  ${CMD} wait deployment ${HCO_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="1200s"
+  ${CMD} wait deployment ${HCO_WH_DEPLOYMENT_NAME} --for condition=Available -n ${HCO_NAMESPACE} --timeout="1200s"
+
+  Msg "operator conditions during upgrade"
+  KUBECTL_BINARY=${CMD} INSTALLED_NAMESPACE=${HCO_NAMESPACE} printOperatorCondition "${I_VERSION}"
+  KUBECTL_BINARY=${CMD} INSTALLED_NAMESPACE=${HCO_NAMESPACE} printOperatorCondition "${T_VERSION}"
+
+  ./hack/retry.sh 30 60 "${CMD} get ${HCO_SUBSCRIPTION} -n ${HCO_NAMESPACE} -o yaml | grep currentCSV   | grep v${T_VERSION}"
+  ./hack/retry.sh  2 30 "${CMD} get ${HCO_SUBSCRIPTION} -n ${HCO_NAMESPACE} -o yaml | grep installedCSV | grep v${T_VERSION}"
+
+  Msg "Verify the hyperconverged-cluster deployment is using the new image"
+
+  set -x
+  SEARCH_PHRASE="${OPENSHIFT_BUILD_NAMESPACE}/pipeline"
+  ./hack/retry.sh 60 30 "${CMD} get -n ${HCO_NAMESPACE} deployment ${HCO_DEPLOYMENT_NAME} -o jsonpath=\"{ .spec.template.spec.containers[0].image }\" | grep ${SEARCH_PHRASE}"
+
+  Msg "Wait that cluster is operational after upgrade"
+  timeout 20m bash -c 'export CMD="${CMD}";exec ./hack/check-state.sh'
+
+  # Make sure the CSV is installed properly.
+  Msg "Read the CSV to make sure the deployment is done"
+  CSV=$( ${CMD} get csv -o name -n ${HCO_NAMESPACE} | grep ${T_VERSION})
+  # Make sure the CSV is in Succeeded phase
+  ./hack/retry.sh 90 10 "${CMD} get ${CSV} -n ${HCO_NAMESPACE} -o jsonpath='{ .status.phase }' | grep 'Succeeded'"
+  # Make sure the CSV is in the correct version
+  ./hack/retry.sh 30 10 "${CMD} get ${CSV} -n ${HCO_NAMESPACE} -o jsonpath='{ .spec.version }' | grep ${T_VERSION}"
+
+  echo "----- Pod after upgrade"
+  Msg "Verify that the hyperconverged-cluster Pod is using the new image"
+  ./hack/retry.sh 10 30 "CMD=${CMD} HCO_NAMESPACE=${HCO_NAMESPACE} ./hack/check_pod_upgrade.sh"
+
+  Msg "Verify new operator version reported after the upgrade"
+  ./hack/retry.sh 15 30 "CMD=${CMD} HCO_RESOURCE_NAME=${HCO_RESOURCE_NAME} HCO_NAMESPACE=${HCO_NAMESPACE} TARGET_VERSION=${T_VERSION} hack/check_hco_version.sh"
+
+  Msg "operator conditions after upgrade"
+  KUBECTL_BINARY=${CMD} INSTALLED_NAMESPACE=${HCO_NAMESPACE} printOperatorCondition "${T_VERSION}"
+
+  [[ -n ${found_new_running_hco_pod} ]]
```

Comment:
@dharmit - The test is failing here. where is this variable defined?

Reply:
My bad. Missed removing this when I removed the code where it was defined. :(